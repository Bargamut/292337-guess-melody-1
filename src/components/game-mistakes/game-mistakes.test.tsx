import * as React from 'react';
import * as renderer from 'react-test-renderer';

import GameMistakes from './game-mistakes';

it(`Game Mistakes renders without mistakes`, () => {
  const gameMistakes = renderer.create(<GameMistakes mistakes={0} />);

  expect(gameMistakes).toMatchSnapshot();
});

it(`Game Mistakes renders with 3 mistakes`, () => {
  const gameMistakes = renderer.create(<GameMistakes mistakes={3} />);

  expect(gameMistakes).toMatchSnapshot();
});
