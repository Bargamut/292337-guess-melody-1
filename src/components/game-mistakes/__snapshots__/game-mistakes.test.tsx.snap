// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game Mistakes renders with 3 mistakes 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
</div>
`;

exports[`Game Mistakes renders without mistakes 1`] = `
<div
  className="game__mistakes"
/>
`;
